src/main/java/core/utils/TrabajopolisUtils.java

```diff
@@ -89,11 +87,11 @@ public String value(final By locator) {
      * This method get the result for assert.
      *
      * @param locator - Locator type.
-     * @return value - Value for the assertion.
+     * @return value - Text of the locator.
      */
     public String message(final By locator) {
```

Comment:
Every method must init with a verb, change the name please.

Reply:
done